web-ui/components/Assets/modals/qckDepositModal.tsx

```diff
@@ -24,7 +26,7 @@ import { useState, useMemo, useEffect } from 'react';
 import { ChooseChain } from '@/components/react/choose-chain';
 import { handleSelectChainDropdown, ChainOption, ChooseChainInfo } from '@/components/types';
 import { useTx } from '@/hooks';
-import { useIbcBalanceQuery } from '@/hooks/useQueries';
+import { useFeeEstimation } from '@/hooks/useFeeEstimation';
```

Comment:
Adding a console error for fee estimations wouldnt solve the issue of not having a fee which would cause the transaction to fail with messaging about a missing fee. The better solution here is to provide a fall back for all of the fee estimate queries.